ui/app/adapters/clients/activity.js

```diff
@@ -4,18 +4,16 @@
  */
 
 import ApplicationAdapter from '../application';
-import { getUnixTime } from 'date-fns';
+import { formatDateObject } from 'core/utils/client-count-utils';
 
 export default class ActivityAdapter extends ApplicationAdapter {
   // javascript localizes new Date() objects but all activity log data is stored in UTC
   // create date object from user's input using Date.UTC() then send to backend as unix
   // time params from the backend are formatted as a zulu timestamp
   formatQueryParams(queryParams) {
     let { start_time, end_time } = queryParams;
-    start_time = start_time.timestamp || getUnixTime(Date.UTC(start_time.year, start_time.monthIdx, 1));
-    // day=0 for Date.UTC() returns the last day of the month before
-    // increase monthIdx by one to get last day of queried month
-    end_time = end_time.timestamp || getUnixTime(Date.UTC(end_time.year, end_time.monthIdx + 1, 0));
```

Comment:
Moved the date formatting logic to the utils file so it could be used elsewhere but it is otherwise unchanged.